spacy/cli/debug_data.py

```diff
@@ -670,6 +671,40 @@ def debug_data(
                 f"Found {gold_train_data['n_cycles']} projectivized train sentence(s) with cycles"
             )
 
+    if "trainable_lemmatizer" in factory_names:
+        msg.divider("Trainable Lemmatizer")
+        trees_train: Set[str] = gold_train_data["lemmatizer_trees"]
+        trees_dev: Set[str] = gold_dev_data["lemmatizer_trees"]
+
+        # This is necessary context when someone is attempting to interpret whether the
+        # number of trees exclusively in the dev set is meaningful.
+        msg.info(f"{len(trees_train)} lemmatizer trees generated from training data.")
+        msg.info(f"{len(trees_dev)} lemmatizer trees generated from dev data.")
+
+        dev_not_train = trees_dev - trees_train
+        if len(dev_not_train) != 0:
+            msg.warn(
+                f"{len(dev_not_train)} lemmatizer trees were found exclusively in the dev data."
+            )
+        else:
+            # Would we ever expect this case? It seems like it would be pretty rare,
+            # and we might actually want a warning?
+            msg.good("All trees in dev data present in training data.")
+
+        if gold_train_data["n_low_cardinality_lemmas"] > 0:
+            msg.warn(
+                f"{gold_train_data['n_low_cardinality_lemmas']} docs with 1 or 0 unique lemmas."
+            )
+        else:
+            msg.good("All training docs meet lemma uniqueness requirements.")
```

Comment:
I don't think users are going to understand "meet lemma uniqueness requirements". I might just leave this part out for the good case?

Reply:
I like the idea to leave it out, I'll do that. I had a hard time wording it in a succinct but descriptive way anyway.

Reply:
@pmbaumgartner : is this done? If so - can you resolve?

Reply:
Whoops, sorry - I totally glanced over this when revisiting this. I still agree it's a bit confusing and a check for a rare case that would require quite a bit of explanation to a user to be useful. I removed this functionality and the corresponding test.

Reply:
I could be mistaken, but I think @adrianeboyd was actually proposing to leave the warning checks in but not print anything for the `good` case?

Reply:
`:me_slapping_forehead:` Let me undo that commit.